Assets/Mirror/Runtime/Messages.cs

```diff
@@ -160,6 +168,19 @@ public class AddPlayerMessage : BytesMessage {}
 
     public class RemovePlayerMessage : EmptyMessage {}
 
+    public class DisconnectMessage : EmptyMessage { }
+
```

Comment:
EmptyMessage (and the other basic messages like stringmessage) need to be inherited from now, correct?
if so, then shouldn't we make them abstract to make sure that no one tries to use stringmessage etc. directly?

Reply:
If we do that we break everyone's code that might be using them.   You can still use EmptyMessage, StringMessage, etc...  with the obsolete methods.

I would say let's make them abstract when we remove the old api